README.md

```diff
@@ -103,6 +103,14 @@ xattr -d com.apple.quarantine <path-to-marksman-bin>
 2. Inside `marksman` folder run `make install`
 3. The binary will be installed under `$HOME/.local/bin` (make sure this folder is in your `PATH`).
 
+### Option 3: use a package manager
+
+```bash
+brew install marksman
+```
+
+> Install Marksman using Homebrew, for Linux and MacOS.
```

Comment:
Perhaps make it into a bullet, e.g.

### Option 3: use a package manager

* Via Homebrew, for Linux and MacOS:
   ```
   brew install marksman
   ```